gradle.properties

```diff
@@ -28,4 +28,4 @@ javaLanguageVersion = 11
 
 # The version of the SPDX license list which is used to import license texts and generate SPDX enums. Must be a valid
 # tag, see https://github.com/spdx/license-list-data/tags.
-spdxLicenseListVersion = 3.23
+spdxLicenseListVersion = 3.24.0
```

Comment:
Maybe worth noting in the commit message that the trailing ".0" is indeed intended, as the upstream Git tag format changed?

Reply:
Another nit: "git" -> "Git"